c7n/resources/s3.py

```diff
@@ -1310,6 +1311,120 @@ def process_bucket(self, bucket):
         return {'Name': bucket['Name'], 'State': 'PolicyRemoved', 'Statements': found}
 
 
+@filters.register('check-public-block')
+class FilterPublicBlock(Filter):
```

Comment:
Attempt at adding a filter but I kind of want this to return like the other things in the augment table. I think I'll also need to enrich it with the bucket name and creation date.